generate-addons.js

```diff
@@ -68,11 +69,20 @@ fs.readdir(specDir, (err, files) => {
       if (hasAlphaVersion) {
         sv.push("alpha");
       }
+      if (preferredVersions[file]) {
+        sv = preferredVersions[file];
+      }
       sv.unshift("latest");
       supportedVersions[file] = sv
     }
   });
 
+  supportedVersions.kubernetes = preferredVersions.kubernetes;
+  preferredVersions.kubernetes.unshift("latest");
```

Comment:
i don't get why doing `unshift` at this point, `preferredVersions.kubernetes` is not used after this line. unless you meant to use `supportedVersions`?

Reply:
i guess modifying `preferredVersions` would also modify `supportedVersions`, but it's less clear this way

Reply:
I'll change to `supportedVersions`